File: Code Studio 1.3/source/dat/main.c
/**
 * Programm....: $
 * Entwickler..: $
 * Version.....: $
 * Datum.......: $
 * Status......: $
 * ToDo........: $
 *               $
 *               $
 */$

/**
 * Praeprozessor
 * Bibliotheken-Einbindung fuer Ein- und Ausgaben (scanf, printf)
 */
 #include <stdio.h>                    /* Standard-Input/Output */
$
/**
 * Funktion..: $
 * Status....: $
 */$
 int i$()
 {
     char sAuswahl[5];                 /* Variable deklarieren */
     do
     {
         printf("\n\t$\n");$
         printf("\n\t< 0 > Beenden\n");
         printf("\n\t----> Auswahl: ");
         scanf("%s", &sAuswahl[0]);    /* Eingabe */
         fflush(stdin);                /* Tastaturpuffer leeren */
         switch(sAuswahl[0])
         {$
             case '0':
             {
                 break;
             }
             default:
             {
                 break;
             }
         }
     }while(sAuswahl[0] != '0');       /* beenden */
     return 0;
 }

/**
 * Funktion..: Steuerprogramm
 */
 int iController()
 {
     i$(); /* Rufe Funktion im Controller auf */
     return 0;
 }

/**
 * Funktion..: Hauptprogramm
 */
 int main()
 {
     iController(); /* Rufe Controller auf */
     return 0;
 }

/**
 * Programmende
 */$

File: Code Studio 1.3/source/dat/main.java
/**
 * Programm....: $
 * Entwickler..: $
 * Version.....: $
 * Datum.......: $
 * Status......: $
 * ToDo........: $
 *               $
 *               $
 */$

/**
 * Packages
 * Bibliotheken-Einbindung fuer Ein- und Ausgaben (print, nextLine)
 */
 import java.util.Scanner;             // Verschiedene nuetzliche Klassen
$
/**
 * Klasse
 */
 public class $
 {$
     /**
      * Funktion..: $
      * Status....: $
      */$
      public void $()
      {
          Scanner scan = new Scanner(System.in); //Eingabe
          String sEingabe;
          char cAuswahl = 0;
          do
          {
              System.out.println("\n\t$\n");$
              System.out.println("\t< 0 > Beenden\n");
              System.out.print("\t----> Auswahl: ");
              sEingabe = scan.nextLine();
              if(sEingabe.length() == 1)
              {
                  cAuswahl = sEingabe.charAt(0); // Zeichen einlesen
              }
              switch(cAuswahl) 
              {$
                  case '0':
                  {
                      break;
                  }
                  default:
                  {
                      break;
                  }
              }
          }while(cAuswahl != '0');     // beenden
      }

     /**
      * Funktion..: Steuerprogramm
      */
      public static void Controller()
      {
          // Erzeuge eine Instanz aus der Klasse $
          $ obj$ = new $();
          // Rufe die Methode der erzeugten Instanz auf
          obj$.$();
      }

     /**
      * Funktion..: Hauptprogramm
      */
      public static void main(String[] args)
      {
          Controller(); // Rufe Controller auf
      }
 }

/**
 * Programmende
 */$
